packages/taco/test/conditions/base/json-api.test.ts

```diff
@@ -7,33 +7,18 @@ describe('JSONPath Validation', () => {
     const invalidPath = '$.store.book[?(@.price < ]';
     const result = jsonPathSchema.safeParse(invalidPath);
     expect(result.success).toBe(false);
-    if (!result.success) {
-      expect(result.error.errors[0].message).toBe(
-        'Invalid JSONPath expression',
-      );
-    }
```

Comment:
Sorry for not being more clear. I meant to keep the `expect` test but remove the `if` i.e. keep only
```typescript
      expect(result.error.errors[0].message).toBe(
        'Invalid JSONPath expression',
      );
```